src/corona_hakab_model/supervisor.py

```diff
@@ -274,7 +271,7 @@ def get(self, manager: "manager.SimulationManager"):
         pass
 
     def snapshot(self, manager: "manager.SimulationManager"):
-        if manager.current_step % self.interval == 0:
+        if manager.current_step > 0 and manager.current_step % self.interval == 0:
```

Comment:
Why skip 0? It is the day of the infection. If we want to snapshot it you are disabling this ability

Reply:
I thought that 0 won't be so informative, so I skipped it.. I added again in the latest commit